generators/docker-compose/templates/_docker-compose.yml

```diff
@@ -14,11 +14,24 @@ services:
             service: kafka
 <%_ } _%>
 <%_ if (gatewayNb + microserviceNb > 0) { _%>
+    <%_ if (serviceDiscoveryType == 'eureka') { _%>
```

Comment:
@xetys You are right and I must say that I have been influenced by all those bad `==` in EJS templates (I used to think that triple equals were not supported in templates), we should get rid of all of those once and for all but maybe not today.